### ISSUE TITLE
Add and index alternate titles in Databases.

### ISSUE THREAD

**Issue by linds37** [ualbertalib/discovery#747]
Hi. This record e-CPS (https://search.library.ualberta.ca/catalog/3774730) leads to an error message

In the Classic Catalogue, when you look up the title e-CPS, the edition that the U of A has access to has a new title: RxTx [electronic resource]. It has  the added entry e-CPS in the 730 field.
here's the Classic Catalogue record: http://www.library.ualberta.ca/permalink/opac/3571716/WUAARCHIVE

**Comment by redlibrarian** [comment]
Hi @linds37 - this item didn't just get a new title, it's a completely new record. The old record (3774730) was removed from Symphony, but is still in the Blacklight index (as we don't have nightly updates going yet). This will be removed next time we do an update, and when we go live this shouldn't be a problem, as long as our updates are happening regularly. / @sjshores @nmacgreg @henryzhang87

**Comment by linds37** [comment]
Thanks. The problem for the Health Sciences staff is that when they look up CPS, or e-CPS, none of the records with that title lead them to the new edition. 
They and their clients don't know to click on RxTx (even though it comes up as the 7th hit in the Books & more bento box). They need a "see" reference based on the 730 field. Can that happen?

**Comment by linds37** [comment]
and, if you just search for CPS (without the e- , which I think most users would do), the RxTx title doesn't come up anywhere in sight

**Comment by redlibrarian** [comment]
@linds37 Yes, I think I can add the 730 to the title search index.

**Comment by linds37** [comment]
awesome!! Can you include both CPS and e-CPS ?

**Comment by redlibrarian** [comment]
The way the Blacklight searching works, CPS should find CPS and e-CPS, once I have that field indexed.

**Comment by linds37** [comment]
excellent!

Lindsay Johnston, Public Services Manager & Subject Librarian
Rutherford Humanities & Social Sciences Library
<http://guides.library.ualberta.ca/rutherford> and Weir Law Library
<http://guides.library.ualberta.ca/law>
1-01 Rutherford Library South
University of Alberta
Edmonton, Alberta
T6G 2J8
780.492.0598

lindsay.johnston@ualberta.ca

**Comment by redlibrarian** [comment]
Actually, @linds37 This seems to already be working... when I search for e-CPS, the RxTx book is 13th in the list of results (below things that have e-CPS or CPS in the main title). Just **CPS** doesn't bring it up, because it's too general, but **CPS pharmacists** finds it. I'm not sure what I can do to improve the ranking of something like that, as the algorithm does find the required volume.... thoughts?

**Comment by linds37** [comment]
I think the RxTx record needs to have CPS in the 730 field

You can use the subject headings in the CPS or e-CPS field to find it easily, though.

**Comment by linds37** [comment]
Is there a way to make RxTx come up in the Databases bento box when you search for CPS (or e-CPS)?

**Comment by sjshores** [comment]
Can you just create it as a database in Libguides?

Sandra

**Comment by linds37** [comment]
I just checked -- RxTx is already in there as a database in Libguides. I just tried creating a see reference "database entry" in Libguides to see what would happen (probably shouldn't have, but I wanted to try it) -- nothing has changed yet in Blacklight, but I don't know when to expect the change to happen.

**Comment by linds37** [comment]
Blacklight is not showing the new database I created (E-CPS see RxTx) -- maybe i did it wrong. One other thing to note -- Blacklight shows the html coding that is in the descriptor field for the database assets in Libguides.

**Comment by redlibrarian** [comment]
@linds37 The databases haven't been updated yet - I'll send an email when that happens. The HTML will also be stripped from the descriptions in the next release (which should be out today or tomorrow).

**Comment by redlibrarian** [comment]
@linds37 The databases have been reindexed, and searching for **E-CPS** brings up your see reference in second place: https://search.library.ualberta.ca/results?utf8=%3F&q=E-CPS

**Comment by theLinkResolver** [comment]
@linds37 I've added CPS to the 730 field in the catalogue record for RxTx. It's definitely worthwhile if that's how our users think of this content. @redlibrarian when would I expect to be able to test the effect of this? (I'm still trying to get a grasp on the flow of things)

**Comment by redlibrarian** [comment]
@linds37 @theLinkResolver A change to symphony should be visible tomorrow or Friday with the new release. AFAICT, the database version is working properly.

**Comment by redlibrarian** [comment]
@linds37 @theLinkResolver The new index should be available in search-test. Not sure if this change is there (still unclear how we're handling updates right now), but it's worth a check...

**Comment by linds37** [comment]
I still don't see the CPS see RxTx entry (no e-  ). Hopefully it will be
picked up soon.
- Lindsay

Lindsay Johnston, Public Services Manager & Subject Librarian
Rutherford Humanities & Social Sciences Library
<http://guides.library.ualberta.ca/rutherford> and Weir Law Library
<http://guides.library.ualberta.ca/law>
1-01 Rutherford Library South
University of Alberta
Edmonton, Alberta
T6G 2J8
780.492.0598

lindsay.johnston@ualberta.ca

**Comment by redlibrarian** [comment]
@linds37 Maybe I'm confused about what's supposed to be appearing. This search: https://search-test.library.ualberta.ca/results?utf8=%3F&q=CPS

has **e-CPS see RxTx** as the first result. Is that not what we're looking for?

**Comment by linds37** [comment]
I also added a database to Libguides that's called just CPS see RxTx -- was
hoping to see them both on the list. That's how Ernie had it set up in the
old database of databases. It seems that the medial community is going to
take a long time to adjust to the new name, since it was called CPS for so
long.
- Lindsay

Lindsay Johnston, Public Services Manager & Subject Librarian
Rutherford Humanities & Social Sciences Library
<http://guides.library.ualberta.ca/rutherford> and Weir Law Library
<http://guides.library.ualberta.ca/law>
1-01 Rutherford Library South
University of Alberta
Edmonton, Alberta
T6G 2J8
780.492.0598

lindsay.johnston@ualberta.ca

**Comment by redlibrarian** [comment]
@linds37 Maybe we need to hash this out in person. I'm unclear what you're expecting to see. If you search for **CPS**, the **CPS see RxTx** database is coming up. Are you expecting to see both **CPS see RxTx** AND **RxTx**? When you search **e-CPS** you get the same two results, with **CPS see RxTx** the second result. Or are you expecting to **RxTx** when you search for any of **CPS**, **e-CPS** or **RxTx**? Thanks. Let me know if you want to meet in person to hash this out.

**Comment by linds37** [comment]
this one: expecting to see *RxTx* when you search for any of *CPS*, *e-CPS*
 or *RxTx*?
we can talk in person!

**Comment by redlibrarian** [comment]
@linds37 OK, thanks. I think I get it now, but it will be good to clarify face-to-face.

**Comment by theLinkResolver** [comment]
@linds37 I added CPS as a 730 to the database *catalogue* record, which has the effect of getting that record in the search results for "cps". That said, the database records from LibGuides are going to be the preferred access point for databases, and that's really the focus of this thread. But I wanted to acknowledge that I added it to the record as you suggested! 

(I'm not sure what will ultimately happen to the corresponding catalogue records for databases. If they are to disappear or be suppressed, I hope there will be efforts to transfer any important details to the LibGuides database entries, first.)
